### DISCUSSION TITLE
is somebody MISUSING this code in my webmail account? I never had errors before this...

### DISCUSSION THREAD

**Discussion by syfy2021** [jquery-form/form#594] [Q&A]
I use a secure email server for my non profit. this past year someone has been constantly breaking into my emails and resetting my passwords. I usually have 2FA setup whenever possible, but because im constantly having to reset everything because so many things are open source including 2FA in some instances, this person always finds a work around? can somebody take a look at this picture and tell me if this javascript is being misused? Not too long ago, he got into mailspring when i started using it, and i saw that he had the github page open for mailspring and had downloaded the mapiproxy info... im not a coder so I am not sure about why or what any of this would be used for. all i know is he is constantly trying to lock me out of my emails and im so tired of it. can somebody help me understand whats going on? Thank you!
 
![my email is compromised because of this javascript from github used by suspect #1 ](https://user-images.githubusercontent.com/86669212/123809297-56f50100-d8a6-11eb-9784-8fdd6eaf5be7.png)

**Comment by kevindb** [comment]
I'm sorry for your trouble, but I don't think it relates to jQuery Form. I cannot help you.
